src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -746,5 +752,40 @@ public static ScreenshotEvent onScreenshot(BufferedImage image, File screenshotF
         MinecraftForge.EVENT_BUS.post(event);
         return event;
     }
+    
+    public static TransformType currentTransformType;
+    
+    public static List<List<BakedQuad>> sortQuadsByIndex(IBakedModel model)
+    {
+        List<List<BakedQuad>> layers = new ArrayList();
+        List<BakedQuad> quads = new ArrayList();
+        
+        for (EnumFacing dir : EnumFacing.values())
+        {
+            quads.addAll(model.getQuads(null, dir, 0));
+        }
+        quads.addAll(model.getQuads(null, null, 0));
+        
+        for (BakedQuad quad : quads)
+        {
+            int index = quad.hasTintIndex() ? quad.getTintIndex() : 0;
+            try { layers.get(index); } catch (IndexOutOfBoundsException e) { layers.add(index, new ArrayList()); }
+            
+            layers.get(index).add(quad);
+        }
+        return layers;
+    }
+    
+    public static void renderItemLayerPre(ItemStack stack, int index)
+    {
+        MinecraftForge.EVENT_BUS.post(new RenderItemLayerEvent.Pre(stack, index, currentTransformType));
+        Tessellator.getInstance().getBuffer().begin(7, DefaultVertexFormats.ITEM);
```

Comment:
You should use the constant `GL11.GL_QUADS` instead of `7` just so someone reading the code knows what it means.